config/default/filter.format.basic_html.yml

```diff
@@ -5,8 +5,13 @@ dependencies:
   config:
     - core.entity_view_mode.media.full
     - core.entity_view_mode.media.stanford_image_large
+    - core.entity_view_mode.media.stanford_image_large_scaled
     - core.entity_view_mode.media.stanford_image_medium
-    - core.entity_view_mode.media.thumbnail
```

Comment:
Definitely dont remove the thumbnail style. This could break the look of existing sites very quickly

Reply:
@heatherdesigns, to respond to your questions:

- For the float issue in IE, it's possible to do a `overflow:hidden` on the content area but that could have side-effects for other content. The default alignment option is `center`... so it seems unlikely that people would intentionally change that for full-width images. I dunno, I'm feeling like recommending we leave it alone, especially since it's only an issue in IE11.
- To Mike's point, we're only removing the thumbnail style from the WYSIWYG options... removing it from the whole codebase would break some default views .
- I believe that any files already created in the WYSIWYG with previous image styles (like thumbnail) won't be changed, but we could test this by finding a piece of existing content, resaving it, and making sure that the images don't break.

Reply:
@bryanbraun 
Confirmed: Thumbnails do not break! :tada:
Medium images will resize from 220px to 360px.